src/base/projects/Header.svelte

```diff
@@ -222,23 +198,9 @@
 </style>
 
 <header>
-  <div class="commit">
-    {#if commit === project.head}
-      <div class="branch">
-        {project.meta.defaultBranch}
-      </div>
-      <div class="hash">
-        {utils.formatCommit(commit)}
-      </div>
-    {:else}
-      <div class="hash desktop">
-        {commit}
-      </div>
-      <div class="hash mobile">
-        {utils.formatCommit(commit)}
-      </div>
-    {/if}
-  </div>
+  <BranchSelector {branches} {project}
```

Comment:
The branch selection logic got too complicated to be inlined here, so I extracted it.